requirements/requirements.txt

```diff
@@ -41,6 +42,7 @@ gunicorn>=22.0.0
 celery==5.4.0
 flower==2.0.1
 redis==5.2.1
+django-redis==5.4.0
```

Comment:
Is this required to do tasks with parameters? If so, let's update the test task to use it.

Reply:
We think tasks with parameters will work without this - I will try without and put back in if we need it